### PR TITLE
Correct Swift region references in Administration Guide (bsc#1105132)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/bk_openstack_admin.xml [hunk 1]
```diff
@@ -13263,9 +13260,7 @@ seeds the Python pseudo-random number generator.</para>
           <step>
             <para>The ring builder assigns each partition replica to the device that
 requires most partitions at that point while keeping it as far away
-as possible from other replicas. The ring builder prefers to assign a
-replica to a device in a region that does not already have a replica.
```
Comment (matthewoliver) [inline xml/bk_openstack_admin.xml]: Why are we removing references to the swift regions? this is true, and a region in Swift isn't necessarily equivalent to regions in keystone or nova.

File: xml/bk_openstack_admin.xml [hunk 2]
```diff
@@ -13206,10 +13206,7 @@ partitions have four replicas and 80 percent have three replicas.</para>
         <para>The replica count is adjustable. For example:</para>
         <screen>$ swift-ring-builder account.builder set_replicas 4
 $ swift-ring-builder account.builder rebalance</screen>
-        <para>You must rebalance the replica ring in globally distributed clusters.
-Operators of these clusters generally want an equal number of replicas
-and regions. Therefore, when an operator adds or removes a region, the
-operator adds or removes a replica. Removing unneeded replicas saves on
+        <para>Removing unneeded replicas saves on
 the cost of disks.</para>
```
Comment (matthewoliver) [inline xml/bk_openstack_admin.xml]: removing replicas do save on cost of disks but you loose durability.

Comment (csymons-suse) [reply xml/bk_openstack_admin.xml]: Please suggest alternative content for removing/not removing replicas.

File: xml/bk_openstack_admin.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/bk_openstack_admin.xml]: Thank you for the comments. Would you please suggest more accurate content?

**Pull request merged**